starter_code/index.html

```diff
@@ -4,112 +4,178 @@
-    <link
-      href="https://use.fontawesome.com/releases/v5.0.1/css/all.css"
-      rel="stylesheet"
-    />
-
-    <!-- link your styles -->
+    <link href="https://use.fontawesome.com/releases/v5.0.1/css/all.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>    <!-- link your styles -->
+    <link rel="stylesheet" href="https://i.icomoon.io/public/temp/fdd0c934da/UntitledProject/style-svg.css">
+    <script defer src="https://i.icomoon.io/public/temp/fdd0c934da/UntitledProject/svgxuse.js"></script>
     <link rel="stylesheet" href="./stylesheets/style.css" />
     <title>Where work happens | Slack</title>
   </head>
 
+<!-- BODY -->
+
   <body>
+
+    <!-- NAVBAR -->
+    
     <nav>
-      <img src="./images/slack-logo.png" alt="slack_logo" />
+      <img class="logo" src="./images/slack-logo.png" alt="slack_logo" />
+      <svg class="icon icon-menu navicon"><use xlink:href="#icon-menu"></use></svg>
       <ul>
-        <li>Why Slack?</li>
-        <li>Pricing</li>
-        <li>About Us</li>
-        <li>Your Workspaces</li>
+        <li><a class="li" href="#">Why Slack?</a></li>
+        <li><a class="li" href="#">Pricing</a></li>
+        <li><a class="li" href="#">About Us</a></li>
+        <li><a class="li-border" href="#">Your Workspaces</a></li>
       </ul>
     </nav>
-    <header>
-      <img src="./images/home_talking.png" alt="home_work_happening" />
-      <h1>Where Work Happens</h1>
-      <p>
-        When your team needs to kick off a project, hire a new employee, deploy
-        some code, review a sales contract, finalize next year's budget, measure
-        an A/B test, plan your next office opening, and more, Slack has you
-        covered.
-      </p>
-      <button type="button" name="button">GET STARTED</button>
-      <p>Already using Slack? <a href="#">Sign in</a></p>
+    
+    <!-- HEADER -->
+    
+    <header class="hero">
+      <div class="container-fluid">
+        <div class="row">
+          <div class="col-12 col-lg-6 order-0 order-lg-1">
+            <h1>Where Work Happens</h1>
+            <p>
+              When your team needs to kick off a project, hire a new employee, deploy
+              some code, review a sales contract, finalize next year's budget, measure
+              an A/B test, plan your next office opening, and more, Slack has you
+              covered.
+            </p>
+            <input type="text" placeholder="Email address">
+            <a href="#" class="button purple">GET STARTED</a>
+            <p>Already using Slack? <a href="#">Sign in</a></p>
```

Comment:
```suggestion
            <div><p>Already using Slack?</p> <a href="#">Sign in</a></div>
```
Cuidado porque envolviste el ancle dentro de un párrafo, recuerda que el párrafo no se comporta como un _div_ o un _section_, si no que indica que va un texto.

Para resolver esta, podrías envolver tanto el párrafo con el texto junto con el _ancle_ dentro de un _div_ y le asignaría una clase, la cual con flexbox la justificaría en el centro.